1

```diff
@@ -0,0 +1 @@
+1remme.economy_enabled1remme.settings.swap_comission1remme.settings.swap_comission1remme.settings.swap_comission1remme.settings.swap_comission1111111remme.economy_enabled1remme.economy_enabled111111remme.economy_enabled1remme.settings.swap_comission1remme.settings.swap_comission1remme.settings.swap_comission1remme.settings.swap_comission1111111remme.economy_enabled1remme.economy_enabled111111remme.economy_enabled1remme.settings.swap_comission1remme.settings.swap_comission1remme.settings.swap_comission1remme.settings.swap_comission11111remme.economy_enabled1remme.economy_enabled111111remme.economy_enabled1remme.settings.swap_comission1remme.settings.swap_comission1remme.settings.swap_comission1remme.settings.swap_comission1111111remme.economy_enabled1remme.economy_enabled111111remme.economy_enabled1remme.settings.swap_comission1remme.settings.swap_comission1remme.settings.swap_comission1remme.settings.swap_comission1111111remme.economy_enabled1remme.economy_enabled111111remme.economy_enabled
```

Comment:
Why this file is presented in the pull request?

Reply:
Done. File deleted